proteus/Transport.py

```diff
@@ -6504,6 +6509,8 @@ def initialize(self,
                     if ts.dofMap.nDOF_all_processes != par_N:
                         mixed=True
                 par_nghost = 0
+                subdomain2global = trialSpaceDict[0].dofMap.subdomain2global
+                max_dof_neighbors= trialSpaceDict[0].dofMap.max_dof_neighbors
                 log("Allocating ghosted parallel vectors on rank %i" % comm.rank(),level=2)
```

Comment:
This is where I would have expected the ParVec...TH class to get used, but I don't see it here.